src/styles/blocks/card.scss

```diff
@@ -0,0 +1,81 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid $color-border-card;
+  border-radius: 5px;
+  background-color: $color-bg-card;
+  padding: 32px 16px 16px;
+
+  &__img {
+    margin: 0 auto 40px;
```

Comment:
your margins should be organized, so move this margin to the bottom as margin-top for title
![image](https://github.com/mate-academy/layout_product-cards/assets/104684446/b5acfb9c-b2de-4ef6-b2f9-459336ebdf41)